### ISSUE TITLE
no jump to higher zoom level (z15)

### ISSUE THREAD

**Issue by Zinnoxid** [Raruto/leaflet-elevation#20]
```html
<!-- D3.js -->
<script src="https://unpkg.com/d3@4.13.0/build/d3.min.js" charset="utf-8"></script>

<!-- Leaflet (JS/CSS) -->
<link rel="stylesheet" href="https://unpkg.com/leaflet@1.3.2/dist/leaflet.css" />
<script src="https://unpkg.com/leaflet@1.3.2/dist/leaflet.js"></script>

<!-- leaflet-geometryutil -->
<script src="https://unpkg.com/leaflet-geometryutil@0.9.1/src/leaflet.geometryutil.js"></script>

<!-- leaflet-distance-marker -->
<link rel="stylesheet" href="https://unpkg.com/@raruto/leaflet-elevation@0.3.9/libs/leaflet-distance-marker.min.css" />
<script src="https://unpkg.com/@raruto/leaflet-elevation@0.3.9/libs/leaflet-distance-marker.min.js"></script>

<!-- leaflet-gpx -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/leaflet-gpx/1.4.0/gpx.js"></script>

<!-- leaflet-gpxgroup -->
<script src="https://unpkg.com/@raruto/leaflet-elevation@0.3.9/libs/leaflet-gpxgroup.min.js"></script>

<!-- leaflet-elevation -->
<link rel="stylesheet" href="https://unpkg.com//@raruto/leaflet-elevation@0.5.8/leaflet-elevation.min.css" />
<script src="https://unpkg.com/@raruto/leaflet-elevation@0.5.8/leaflet-elevation.min.js"></script>

<link rel="stylesheet" href="div/leaflet.zoomdisplay.css" />
<script type="text/javascript" src="div/leaflet.zoomdisplay.js"></script>

<!-- leaflet-locatecontrol -->
<link rel="stylesheet" href="https://unpkg.com/leaflet.locatecontrol@0.67.0/dist/L.Control.Locate.min.css" />
<script src="https://unpkg.com/leaflet.locatecontrol@0.67.0/dist/L.Control.Locate.min.js"></script>
```

I did some tests from version **0.5.0** until **0.6.6**
	
Version **0.5.8** is the only one where the map zooms to **level 15** when I move the mouse pointer inside the chart

**Comment by Raruto** [comment]
I do not understand what is the bug or the unexpected behavior...

Please, can you provide **complete examples**?

**Comment by Zinnoxid** [comment]
I use your example "hoverable .gpx tracks"

I took off pegman, minimap. Added zoomdisplay.

I use this hiking route https://www.openstreetmap.org/relation/157611#map=8/49.390/8.130

After hovering the mouse pointer on the route, the bottom chart appers. But the zoom level of the map do not change to a higher level (15). With version 0.5.8 the map jumps to the higer level, when the pointer is inside the chart.
I tested version 0.5.0 until 0.6.6.

`<!DOCTYPE html>
<html>

<head>
	<title>leaflet-elevation.js</title>
	<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
	<meta name="viewport" content="initial-scale=1.0, user-scalable=no" />
	<style>
		html,
		body,
		#map,
		#elevation-div {
			height: 100%;
			width: 100%;
			padding: 0px;
			margin: 0px;
		}
	</style>

	
	<script src="https://unpkg.com/d3@4.13.0/build/d3.min.js" charset="utf-8"></script>

	
	<link rel="stylesheet" href="https://unpkg.com/leaflet@1.3.2/dist/leaflet.css" />
	<script src="https://unpkg.com/leaflet@1.3.2/dist/leaflet.js"></script>

	
	<script src="https://unpkg.com/leaflet-geometryutil@0.9.1/src/leaflet.geometryutil.js"></script>

	
	<link rel="stylesheet" href="https://unpkg.com/@raruto/leaflet-elevation@0.3.9/libs/leaflet-distance-marker.min.css" />
	<script src="https://unpkg.com/@raruto/leaflet-elevation@0.3.9/libs/leaflet-distance-marker.min.js"></script>

	
	<script src="https://cdnjs.cloudflare.com/ajax/libs/leaflet-gpx/1.4.0/gpx.js"></script>

	
	<script src="https://unpkg.com/@raruto/leaflet-elevation@0.3.9/libs/leaflet-gpxgroup.min.js"></script>

	
	<link rel="stylesheet" href="https://unpkg.com//@raruto/leaflet-elevation@0.6.0/leaflet-elevation.min.css" />
	<script src="https://unpkg.com/@raruto/leaflet-elevation@0.6.0/leaflet-elevation.min.js"></script>
	
	
	<link rel="stylesheet" href="div/leaflet.zoomdisplay.css" />
	<script type="text/javascript" src="div/leaflet.zoomdisplay.js"></script>

	
	<link rel="stylesheet" href="https://unpkg.com/leaflet.locatecontrol@0.67.0/dist/L.Control.Locate.min.css" />
	<script src="https://unpkg.com/leaflet.locatecontrol@0.67.0/dist/L.Control.Locate.min.js"></script>
	

	<style>
		.leaflet-control-locate a {
			background-image: url(https://unpkg.com/@raruto/leaflet-elevation@0.3.9/images/elevation-locate.png);
		}

		.leaflet-control-layers-list {
			max-width: 250px;
			max-height: 25vh;
			overflow: auto;
		}

		.leaflet-tooltip {
			font-weight: 700;
		}

		#map {
			height: 75%;
		}

		#elevation-div {
			height: 25%;
			font: 12px/1.5 "Helvetica Neue", Arial, Helvetica, sans-serif;
		}

		.data-summary {
			font-family: "Lucida Grande", "Lucida Sans Unicode", Verdana, Arial, Helvetica, sans-serif;
			font-size: 12px;
			position: absolute;
			bottom: 0;
			left: 45px;
		}

		.data-summary .summaryvalue {
			font-weight: bold;
		}
		.chart-placeholder {
			top: 20%;
			position: absolute;
			text-align: center;
			left: 0;
			right: 0;
			font-family: "Lucida Grande", "Lucida Sans Unicode", Verdana, Arial, Helvetica, sans-serif;
			font-style: italic;
			opacity: 0.5;
			font-size: 0.9rem;
		}
	</style>

</head>

<body>

	<div id="map"></div>
	<div id="elevation-div" class="leaflet-control elevation"><p class="chart-placeholder"><b>Bewegen Sie den Mauszeiger &uuml;ber eine Route ...</b></p></div>

	<div id="data-summary" class="data-summary"><span class="totlen"><span class="summarylabel">Gesamtl&auml;e: </span><span class="summaryvalue">0</span></span> &mdash; <span class="maxele"><span class="summarylabel">H&ouml;chste Erhebung: </span><span class="summaryvalue">0</span></span>
		&mdash; <span class="minele"><span class="summarylabel">Geringste Erhebung: </span><span class="summaryvalue">0</span></span></div>

	<script>
		var opts = {
			map: {
				center: [49.337278, 7.825249],
				zoom: 5,
				zoomControl: false,
			},
			zoomControl: {
				position: 'bottomright',
			},
			wikimediaLayer: {
				url: 'https://maps.wikimedia.org/osm-intl/{z}/{x}/{y}{r}.png',
				options: {
					maxZoom: 17,
					attribution: 'Map data: &copy; <a href="https://wikimediafoundation.org/wiki/Maps_Terms_of_Use">Wikimedia</a>)',
				},
			},
			otmLayer: {
				url: 'https://{s}.tile.opentopomap.org/{z}/{x}/{y}.png',
				options: {
					maxZoom: 17,
					attribution: 'Map data: &copy; <a href="http://www.openstreetmap.org/copyright">OpenStreetMap</a>, <a href="http://viewfinderpanoramas.org">SRTM</a> | Map style: &copy; <a href="https://opentopomap.org">OpenTopoMap</a> (<a href="https://creativecommons.org/licenses/by-sa/3.0/">CC-BY-SA</a>)',
				},
			},
			osmLayer: {
				url: 'https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png',
				options: {
					maxZoom: 19,
					attribution: 'Map data: &copy; <a href="http://www.openstreetmap.org/copyright">OpenStreetMap</a>',
				}
			},
			satelliteLayer: {
				url: 'http://server.arcgisonline.com/ArcGIS/rest/services/World_Imagery/MapServer/tile/{z}/{y}/{x}',
				options: {
					maxZoom: 18,
					attribution: '&copy; <a href="http://www.esri.com/">Esri</a>',
				},
			},
			layersControl: {
				options: {
					position: 'topleft',
					collapsed: false,
				},
			},
			scaleControl: {
				options: {
					width: 200,
					position: 'bottomright',
					imperial: false,
				},
			},
				
			locateControl: {
				options: {
					position: "bottomright",
				},
			},
			points: {
				options: {
					icon: {
						iconUrl: 'https://unpkg.com/@raruto/leaflet-elevation@0.3.9/images/elevation-poi.png',
						iconSize: [12, 12],
					},
				},
			},
			elevation: {
				options: {
					theme: 'yellow-theme',
					// width: 500,
					detachedView: true,
					elevationDiv: '#elevation-div',
				},
			},
		};

		var tracks = [
			
			"elevation_tracks/blaues_kreuz_157611_overpass_hoehen.gpx",
			
			
			
		];

		var points = [
			{latlng:[49.337278, 7.825249], name:"Johanniskreuz"},
			
		];

		var map = L.map('map', opts.map);

		var baseLayers = {};

		baseLayers["ESRI World Imagery"] = L.tileLayer(opts.satelliteLayer.url, opts.satelliteLayer.options);
		baseLayers["OpenTopoMap"] = new L.TileLayer(opts.otmLayer.url, opts.otmLayer.options);
		baseLayers["OpenStreetMap"] = new L.TileLayer(opts.osmLayer.url, opts.osmLayer.options);
		baseLayers["Wikimedia"] = new L.TileLayer(opts.wikimediaLayer.url, opts.wikimediaLayer.options);

		var controlZoom = new L.Control.Zoom(opts.zoomControl);
		var controlLayer = L.control.layers(baseLayers, null, opts.layersControl.options);
		var controlScale = L.control.scale(opts.scaleControl.options);
		
		var controlLocate = L.control.locate(opts.locateControl.options);

		var routes = L.gpxGroup(tracks, {
			points: points,
			points_options: opts.points.options,
			elevation: true,
			elevation_options: {
				theme: "yellow-theme",
				detachedView: true,
				elevationDiv: '#elevation-div',
				followPositionMarker: true,
				zFollow: 15,
			},
			legend: true,
			distanceMarkers: false,
		});

		map.on('eledata_added eledata_clear', function(e) {
			var p = document.querySelector(".chart-placeholder");
			if(p) {
				p.style.display = e.type=='eledata_added' ? 'none' : '';
			}
		});

		map.on('eledata_added', function(e) {
			var q = document.querySelector.bind(document);
			var track = e.track_info;

			//controlLayer.addOverlay(e.layer, e.name);
			q('.totlen .summaryvalue').innerHTML = track.distance.toFixed(2) + " km";
			q('.maxele .summaryvalue').innerHTML = track.elevation_max.toFixed(2) + " m";
			q('.minele .summaryvalue').innerHTML = track.elevation_min.toFixed(2) + " m";
		});

		controlLayer.addTo(map);
		controlScale.addTo(map);
		controlZoom.addTo(map);
		controlLocate.addTo(map);

		routes.addTo(map);

		map.addLayer(baseLayers["Wikimedia"]);

		map.attributionControl.addAttribution('Map Data: &copy; <a href="https://github.com/adoroszlai">Attila Doroszlai</a> (<a href="https://github.com/adoroszlai/joebed/blob/gh-pages/LICENSE.md">MIT License</a>)');
	</script>

	
</body>

</html>`

**Comment by Raruto** [comment]
There was a wrong check condition alias for the `followMarker / followPositionMarker` options. (introduced in version **0.5.9**,  now fixed in **version 0.6.7**).
